versions.json

```diff
@@ -159,17 +151,9 @@
     "maintenance": "2021-10-19",
     "end": "2023-04-30",
     "codename": "fermium",
-    "alpine-default": "alpine3.11",
+    "alpine-default": "alpine3.14",
```

Comment:
Yeah, the per-release aliases have existed since https://github.com/docker-library/official-images/commit/83817bcd9f5258a7008c62ef2217135b05a314c7 (commit was even yours, @SimenB :joy:), so IMO there's been a pretty reasonable amount of time for users to notice and get used to them, and start using them if they're Alpine-version-sensitive.